src/zenml/zen_stores/local_zen_store.py

```diff
@@ -921,7 +921,7 @@ def get_role_assignments_for_team(
 
     # Pipelines and pipeline runs
 
-    def get_pipeline_run(
+    def get_pipeline_run_wrapper(
```

Comment:
Shouldn't this entire yaml store be deleted?

Reply:
Yeah, I have another branch for that, but didn't go ahead yet since we haven't made a final decision of whether to migrate it or not. If we do migrate, then keeping (and potentially adjusting) the class will be required.